src/main/java/Storage/Storage.java

```diff
@@ -0,0 +1,203 @@
+package Storage;
+
+import java.io.*;
+import java.util.Arrays;
+import Task.TaskList;
+import Task.Task;
+import Task.TaskTypes;
+import Task.DeadlineException;
+import Task.TodoException;
+import Task.EventsException;
+
+public class Storage {
+    private final Parser parser= new Parser();
```

Comment:
```suggestion
    private final Parser PARSER = new Parser();
```

Reply:
I think if you are declaring a constant, it needs to be in caps. Also, don't forget the space before the "=".